src/markdown-pages/build-apps/create-an-ab-test-application/publish.mdx

```diff
@@ -169,9 +169,9 @@ Go to [New Relic](https://one.newrelic.com). Notice that you're not using the `?
 
 <Step>
 
-From the homepage, navigate to **Apps**:
+From the homepage, navigate to **Instant Observability**:
 
-![Navigate to Apps](../../../images/build-an-app/nav-to-apps.png)
+![Navigate to Instant Observability](../../../images/serve-publish-subscribe/nav-to-io.png)
 
 From here, you can see your application under **New Relic One catalog**:
```

Comment:
@alexronquillo These are great suggestions thank you! I'm curious what kind of place the term `catalog` still has in this experience, since we do have a Nerdpack artifact called `catalog` that is purposed around providing Nerdpack metadata for the "catalog". 

https://developer.newrelic.com/build-apps/publish-deploy/catalog/ 

Is there a distinction to made here of "New Relic One catalog" as a named place in the UI that was replaced by "Instant Observability", but that Instant Observability has a "catalog" of resources in a non-proper name sense? 

Would it be ok in some instances to say the Instant Observability catalog so that there is still some connection to this idea of a catalog and that aspect of a Nerdpack?

Reply:
I think you're right that there is a place for "catalog" as it applies to the registry of all published Nerdpacks, the Nerdpack artifact, and the command. I also agree with your distinction and that primarily what we want to change here are references to the New Relic One catalog in the UI.

I'll defer to @jpvajda about whether "Instant Observability catalog" is ok since I haven't really been part of those conversations and could go either way.

Reply:
Example

> In the last lesson, you created catalog information for the A/B test app you've been building throughout this course. Now, it's time to publish your app to the New Relic One catalog and submit those catalog details.

Could be written as?: 

> In the last lesson, you created catalog information for the A/B test app you've been building throughout this course. Now, it's time to publish your app to the Instant Observability catalog and submit those catalog details.

Reply:
@alexronquillo Just pushed another pass on this updating the "New Relic One catalog" references to "Instant Observability"

I've also included in places references to "the Instant Observability catalog" to help connect catalog metadata to I/O. Happy to go back and edit those depending on how @jpvajda weighs in.

Reply:
I've gone through pretty closely and feel like these changes are really solid. Thanks again! Here's another example where "the Instant Observability catalog" feels like the right phrasing since it connects the CLI command / directories in the repo to their UI representation. 

![Screen Shot 2021-09-29 at 2 06 46 PM](https://user-images.githubusercontent.com/20293876/135353090-e2355114-b08c-4765-bcd7-ce59b3a37098.jpg)